en/core-api/advanced-features.md

```diff
@@ -707,25 +911,31 @@ curl -X POST \
 <!-- <TODO: elaborate Subscriptions API> -->
 
 ### Two Clicks Transaction
-You can allow customer to save their card credentials (**except card cvv**) for easier and faster future transactions. Card credentials will be saved securely on Midtrans side. Merchant will only need to store and associate each unique customer with unique `saved_token_id` from initial Charge Response.
+You can allow customer to save their card credentials, **except the CVV number**, for easier and faster future transactions. Midtrans saves the card credentials securely. You have to store and associate each customer with a unique `saved_token_id` from the initial Charge API Response.
 
 <details>
 <summary><b>Sequence Diagram</b></summary>
 <article>
-The overall Two Clicks end-to-end payment proccess can be illustrated in following sequence diagram:
-
+The Two Clicks end-to-end payment process is illustrated in following sequence diagram:
 ![one click sequence diagram](../../asset/image/core_api-sequence_two_clicks.png)
```

Comment:
The image fails to render after the change you made: removing the blank line, try to revert the changes on the blank line.